entity-service/config/detekt/baseline.xml

```diff
@@ -36,5 +36,6 @@
     <ID>TooGenericExceptionCaught:ParsingUtils.kt$e: Exception</ID>
     <ID>TooManyFunctions:EntityService.kt$EntityService$EntityService</ID>
     <ID>TooManyFunctions:Neo4jRepository.kt$Neo4jRepository$Neo4jRepository</ID>
+    <ID>LargeClass:EntityService.kt$EntityService</ID>
```

Comment:
Did you forget to talk about this?

Reply:
Should I add a service for attributes ?
Or we do the refactoring in another PR ?